docs/DeveloperGuide.adoc

```diff
@@ -333,6 +333,69 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+=== Return a Loan feature
+==== Proposed Implementation
+
+Returning a loan is done by setting the corresponding status of the loan to returned.
+
+The steps that have to be done by LoanBook is as follows:
+* Store the current time as the return time of the loan
+* Change the enum of `LoanStatus` to become `Status.RETURNED`
+* Calculate the cost of the loan and display it in the gui as a suggestion for the user.
+
+These steps would change the given loan object, and update the properties of the loan object. The appropriate values stored within the Loan object would therefore change accordingly.
+
+Given below is an example usage scenerio and how the internals of the Loan would behave:
+
+**Step 1.** The user would have to do a search for which loan his would like to return. This will pull up a list of Loans from which the user would be able to select the correct loan.
```

Comment:
*which loan he would like to return
(or "they")

Reply:
OK. Fixed.